app/views/site/_view_gene_expression_title_bar.html.erb

```diff
@@ -5,4 +5,17 @@
       <h3 class="help-block lead">Expression values are shown as the <strong id="selected-consensus"><%= params[:consensus].nil? ? 'mean' : params[:consensus] %></strong> of expression values across all genes for each cell.</h3>
     <% end %>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<script>
+  // Initializes Ideogram for related genes
+
+  document.querySelector('.study-lead').parentElement.innerHTML +=
+    '<div id="related-genes-ideogram-container" style="visibility: hidden; position: absolute; z-index: 1000; width: 100%; background: #FFF;"></div>';
```

Comment:
same comment about static css file.  IIRC correctly David Mohs and Terra are big fans of inline CSS for almost all cases (which makes more sense for Terra given that it is exclusively built out of components), but I think we should be consistent within SCP  of sticking to external CSS files, unless and until we make a decision to migrate the product as a whole over to inline

Reply:
[Done](https://github.com/broadinstitute/single_cell_portal_core/blob/1731b32e9d0f2a7fbd6d89c177b5eebc95ca0641/app/javascript/styles/_relatedGenesIdeogram.scss).  While the code is more elegant, it loses at least as much cohesion.  External CSS shines for styles intended for reuse.  For single-use components like this, the case for external CSS is less clear.  That said, I'll commit to the principle -- you make a good point about consistency.